### ISSUE TITLE
Setter doesn't work.

### ISSUE THREAD

**Issue by justinbmeyer** [canjs/can-view-scope#193]
`<button on:click='this.todo.disabled = false'>allow</button>`

This seems to be trying to set `this.disabled` incorrectly.  

```
Component.extend({
	tag: "todo-create",
	view: `

			<div class="form-group">
				<label>Name</label>
				<input class="form-control"
					on:input:value:bind='this.todo.name'/>
			</div>
		<button on:click='this.todo.disabled = false'>allow</button>
			{{this.todo.disabled}}
			<button disabled:from="todo.disabled"
				class="btn btn-primary">
				{{# if(this.todo.isSaving()) }}
					Creating .... 
				{{ else }}
					Create Todo 
				{{/if}}
			</button> 
				

	`,
	ViewModel: {
		todo: {
			default(){
				return new DefineMap({disabled: true})
			}
		}
	}
});
```

**Comment by cherifGsoul** [comment]
I can not reproduce the issue in this [codepen](https://codepen.io/cherifGsoul/pen/YdJwap) , could it be fixed?

**Comment by chasenlehara** [comment]
Thanks @cherifGsoul, this looks fixed as of can@5.21.5
